code/modules/power/supermatter/supermatter.dm

```diff
@@ -722,7 +722,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			supermatter_anomaly_gen(src, FLUX_ANOMALY, rand(5, 10))
 		if(power > SEVERE_POWER_PENALTY_THRESHOLD && prob(5) || prob(1))
 			supermatter_anomaly_gen(src, GRAVITATIONAL_ANOMALY, rand(5, 10))
-		if((power > SEVERE_POWER_PENALTY_THRESHOLD && prob(2)) || (prob(0.3) && power > POWER_PENALTY_THRESHOLD))
+		if(power > SEVERE_POWER_PENALTY_THRESHOLD && prob(2))
```

Comment:
this is definetly snowflaking the SM, you have a probability of 0.3 of spawning the pyro anomaly if the power is over 5000 and you still want to nerf it. Don't go over 5k MeV if you can't handle it, the SM is not a toy to play with

Reply:
It doesn't matter small the chance is, if the slimes spawn they will destroy the SM instinctively. The goal is to not babysit the SM. If you can stabilize the SM between 5000 and 7500, you should be able to capitalize off of it.

Reply:
What you want is for everyone to have to use your SM layout without floors. That is snowflaking, and it's at the expense of people playing the game to play the game.

Reply:
If you don't want to babysit the SM don't go over the Power treshold, a stable SM at 5000 to 7500 is doable but stable doesnt mean safe, you have to stay there otherwise take another job (also since you keep bringing up my setup, well that setup is completely safe, you can go do anything else you want and the SM will never fail)